tests/integration/cgroups.bats

```diff
@@ -110,6 +110,17 @@ EOF
 
     runc run -d --console-socket $CONSOLE_SOCKET test_cgroups_permissions
     [ "$status" -eq 0 ]
+    if [ "$CGROUP_UNIFIED" != "no" ]; then
+        if [ -n "${RUNC_USE_SYSTEMD}" ] ; then
```

Comment:
nit:
```go
if [ "$CGROUP_UNIFIED" != "no" -a -n "${RUNC_USE_SYSTEMD}" ]; then
```

Reply:
I think no, because there is `else` for `if [ -n "${RUNC_USE_SYSTEMD}" ] ; then`.

Reply:
sorry, missed it